packages/fluentui/styles/test/mergeThemes/mergeComponentStyles-test.ts

```diff
@@ -225,47 +227,25 @@ describe('mergeComponentStyles', () => {
       });
 
       test('are flat for recursive merge', () => {
-        const target = withDebugId(
-          {
-            root: {
-              a: 'tA',
-            },
-          },
-          'target',
-        );
-        const source1 = withDebugId(
-          {
-            root: {
-              a: 'tB',
-            },
-          },
-          'source1',
-        );
-        const source2 = withDebugId(
-          {
-            root: {
-              a: 'tC',
-            },
-          },
-          'source2',
-        );
+        const target = withDebugId({ root: { a: 'tA' } }, 'target');
+        const source2 = withDebugId({ root: { a: 'tB' } }, 'source');
```

Comment:
nit: Why is this `source2`, not just `source`?

Reply:
Copy-paste issue, fixed 👍